src/config/index.ts

```diff
@@ -145,6 +145,7 @@ export const PUBLIC_REGISTER_NAME = "#public_register_name";
 export const ROLE_AND_RESPONSIBILITIES = "#role_and_responsibilities";
 export const CONTACT_FULL_NAME = "#contact_full_name";
 export const CONTACT_EMAIL = "#contact_email";
+export const CREATED_DATE = "#createdDate-createdDateDay";
```

Comment:
> Suggest following existing convention here in terms of naming, so split names using underscores and dashes. E.g. #created_date-created_date_day (I think that matches the pattern used for example by DATE_OF_BIRTH).

this is field name
we use CamelCase for field name

Reply:
@mwestacott - this field name was done in the previous PR for the Trust details screen.

Have a chat with Tom and please come to a decision on if we should change this. However if you both agree then I suggest we do that in a new Jira and so a new PR (so please resolve this comment)

Reply:
I ca not see consistent problem here.
In your implementation you dependant on form field name, because form field must be equal session field name.
In our implementation, we broke this link. Son now form filed can be different than session field and have more readable view.

Here just text, hash to field name. We still point on field name in form.
Nothing broken

Reply:
That field should represent an id input on your trust page, used to point to the right field on the right page, but it seems not present? probably I missed something

Reply:
If it was working ignore me, but If I am reading correctly probably it should have been `createdDate` and we need to add an id on the field in your html page

Reply:
@mouhajer-ch - I have just retested it and the link works

Reply:
Thanks @SteveHicksCH for double checking 👍